### PR TITLE
fix: fix regex in wordcount

### PR DESCRIPTION
**Pull request by ADSWT518**
The original regex will match `openany` or `openright` option incorrectly.

### CODE REVIEW

File: Compile.bat [hunk 1]
```diff
@@ -77,7 +77,7 @@ goto :EOF
 	set found=0
 	setlocal enabledelayedexpansion
 
-	findstr "\\documentclass\[[^\[]*en" %THESIS%.tex > nul
+	findstr "\\documentclass\[[^\[]*lang\s*=\s*en" %THESIS%.tex > nul
```
Comment (LogCreative) [inline Compile.bat]:
```suggestion
	findstr \R \C:"\\documentclass\[[^\[]*lang *= *en" %THESIS%.tex > nul
```

`\s` may not work for Windows `findstr`, you can try the test case on Windows:
```latex
\documentclass[lang = en]{sjtuthesis}
```

Simply replace `\s` to ` ` is not enough, since ` ` works like `|` for `findstr`[^1]. You can try the test case
```latex
\documentclass[lang=zh]{sjtuthesis}
```

Use `\C:` switch to take the space input and then use regrex switch `\R` to interpret. This may be the solution.

[^1]: https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/findstr

**Review by LogCreative — CHANGES_REQUESTED** [review]
Thanks for your contribution! An edge case may need to be solved before merging.

**Comment by ADSWT518** [comment]
Oh! Thanks for your correction! I'm sorry that I didn't test `Compile.bat` before committing, since I didn't have LaTeX on Windows :crying_cat_face:

**Review by LogCreative — APPROVED** [review]
Thanks for your contribution!
